docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml

```diff
@@ -1040,6 +1041,29 @@
           on appropriate nodes.</para>
         </sect4>
       </sect3>
+
+      <sect3 id="ENV_values_yaml">
+        <title>Environment Values</title>
+
+        <para>You can define Environment Variables in a YAML file. The
+        environment values are defined under the
+        <emphasis>global.env:</emphasis> portion of the delivered values.yaml
+        as illustrated below.</para>
+
+        <programlisting>global:
+  env:
+  - name: SMTPserver 
+  - value: mysmtpserver</programlisting>
+
+        <para>The global.env section of the delivered values file adds default
```

Comment:
question: What does the phrase "delivered values file" mean?

Reply:
referring to the default/stock https://github.com/hpcc-systems/HPCC-Platform/master/helm/hpcc/values.yaml file,
is there a better way to refer to it?

Reply:
perhaps "the supplied values.yaml file" is better?